yarn.lock

```diff
@@ -3449,6 +3435,21 @@ execa@^3.2.0:
     signal-exit "^3.0.2"
     strip-final-newline "^2.0.0"
 
+execa@^4.0.0, execa@^4.0.3:
```

Comment:
this was moved to the correct place; previously it was above the ones that had numbers lower than it